### ISSUE TITLE
Compatibility with Atom 1.12?

### ISSUE THREAD

**Issue by idleberg** [spark/console-panel#3]
Is this package still compatible with Atom 1.12? I'm asking because whenever I add the code to my `package.json`/`main.coffee`, I get dozens of these errors in my console:

```
state-store.js:33 Uncaught (in promise) DOMException: Failed to execute 'put' on 'IDBObjectStore':
    An object could not be cloned.(…)
```

**Comment by suda** [comment]
Hi Jan! Thx for this report. I just released `0.2.1` which should fix this problem.

**Comment by idleberg** [comment]
That was quick, thank you!
